lib/needle.js

```diff
@@ -478,19 +486,25 @@ Needle.prototype.send_request = function(count, method, uri, config, post_data,
     done(err || new Error('Unknown error when making request.'));
   }
 
+  function abort_handler() {
+    out.emit('err', new Error('Aborted by signal.'));
+    request.destroy();
+  }
+
   function set_timeout(type, milisecs) {
     if (timer) clearTimeout(timer);
     if (milisecs <= 0) return;
 
     timer = setTimeout(function() {
       out.emit('timeout', type);
-      request.abort();
+      request.destroy();
```

Comment:
Is the right way of aborting a http.request these days? From what Node.js version is this supported?

Reply:
<img width="924" alt="image" src="https://user-images.githubusercontent.com/11750429/204021922-002d8f80-95d2-49d6-b3e8-3250bcdc051e.png">

Do you think we can remove deprecations?